README.md

```diff
@@ -1,4 +1,4 @@
-![Screenshot](https://raw.githubusercontent.com/cram2/cram/master/CramLogoSmall.png)
+![Screenshot](https://raw.githubusercontent.com/cram2/cram/master/cram_logo_and_readme/CramLogoSmall.png)
```

Comment:
It's called cram_graphics not cram_logo_and_readme, so it's not gonna work.
Actually, can you rename the directory into just "graphics" instead of "cram_graphics" and adjust the link as well?

Reply:
done, as well i added the cram_logo in big for future purpose